.rspec

```diff
@@ -0,0 +1,2 @@
+--color
```

Comment:
You may consider moving this to your home folder. That will apply to all projects.